Px.Utils/Validation/ContentValidation/ContentValidator.cs

```diff
@@ -7,14 +7,14 @@ namespace PxUtils.Validation.ContentValidation
 {
     /// <summary>
     /// Provides methods for validating the content of Px file metadata
-    /// <param name="_filename">Name of the Px file</param>
-    /// <param name="_encoding"> _encoding of the Px file</param>
-    /// <param name="_syntaxConf">Object that stores syntax specific symbols and tokens for the Px file</param>
+    /// <param name="filename">Name of the Px file</param>
+    /// <param name="encoding"> encoding of the Px file</param>
+    /// <param name="syntaxConf">Object that stores syntax specific symbols and tokens for the Px file</param>
     /// </summary>
-    public sealed partial class ContentValidator(string _filename, Encoding _encoding, PxFileSyntaxConf? _syntaxConf = null)
+    public sealed partial class ContentValidator(string filename, Encoding encoding, PxFileSyntaxConf? syntaxConf = null)
     {
-        private readonly string filename = _filename;
-        private readonly Encoding encoding = _encoding;
+        private readonly string filename = filename;
```

Comment:
Private field names should start with _ to be consistent within the project.
private readonly string _filename = filename;